rfd/0079/project-service-draft.md

```diff
@@ -0,0 +1,121 @@
+# Mariposa Project service
+
+The Mariposa (RFD36) Project service will be responsible for maintaining and exposing data about user projects and services. The project service will expose multiple endpoints for interacting with this data, to be used by the Triton CLI as well as other Triton services, including the Convergence service. 
+
+Projects and their services will be wholly versioned such that a change to the project manifest, metadata, or any service associated with the project will result in a new (sha1 sum) tag being created. 
+
+## Data
+
+The Project service will persist data to an appropriate backend (Postgres/Manatee?). Persisted data will represent user projects and their associated services. See:
+
+* [Mariposa service manifest](https://github.com/joyent/rfd/blob/master/rfd/0036/service-manifest.md)
+* [Mariposa project manifest](https://github.com/joyent/rfd/blob/master/rfd/0036/project-manifest.md)
+
+In addition to manifest data, service goal state data will be managed by the Project service, including:
+
+* status - whether a service is stopped, started, or paused
+* scale - the number of containers that should be running for a particular service
+
+## Project and service tagging
+
+A project will be automatically tagged with the sha1 sum of it's parts (project manifest, all service manifests, and any applicable metadata). This will occur upon the creation of the project or any change to it or it's associated services. 
```

Comment:
I mean that the datetime of the change must be recorded, in addition to the sha1.